packages/did-manager/src/abstract-identifier-provider.ts

```diff
@@ -10,6 +10,11 @@ export abstract class AbstractIdentifierProvider {
     context: IAgentContext<IKeyManager>,
   ): Promise<Omit<IIdentifier, 'provider'>>
 
+  abstract updateIdentifier(
```

Comment:
This method can be made optional:

```suggestion
  abstract updateIdentifier?(
```

Reply:
Agreed. Included & removed extra throwing from other providers.